### DISCUSSION TITLE
Need help with Student Developer Package

### DISCUSSION THREAD

**Discussion by arishaprasain** [community/community#35924] [GitHub Education]
I am trying to apply for Student Developer Pack but go the following before submitting:
Your application cannot be reviewed until you fix the following:
Your school is a GitHub Campus partner. Please contact your school administrator to have the correct benefits applied to your GitHub account.

But my campus (Pulchowk Campus) IS a GitHub Campus Partner. What should I do?
![Screenshot (220)](https://user-images.githubusercontent.com/105920260/195387847-42fd32aa-e5b9-4077-87c1-51408a5559a5.png)

**Comment by MitkoVtori** [comment]
You should contact the school administrator to approve something about your acc.

**Comment by shubham8550** [comment]
i guass u should retry
